src/System.Memory/tests/ReadOnlySpan/SequenceCompareTo.int.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class ReadOnlySpanTests
+    {
+        [Fact]
+        public static void ZeroLengthSequenceCompareTo_Int()
+        {
+            var a = new int[3];
+
+            var first = new ReadOnlySpan<int>(a, 1, 0);
+            var second = new ReadOnlySpan<int>(a, 2, 0);
+            int result = first.SequenceCompareTo<int>(second);
+            Assert.Equal(0, result);
+        }
+
+        [Fact]
+        public static void SameSpanSequenceCompareTo_Int()
+        {
+            int[] a = { 4, 5, 6 };
```

Comment:
nit: I realize we have similar code elsewhere, but can we use values that are larger than byte (i.e. larger than 255), and that can only fit into an integer, going forward?

Random values somewhere between Int16.MaxValue (32767) and Int32.MaxValue (2147483647) would be best.

See recently added SequenceEqual.long as an example: https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/Span/SequenceEqual.long.cs#L34